content/docs/command-reference/add.md

```diff
@@ -378,3 +380,49 @@ system that can handle it), they can use `dvc pull` as usual:
 A       data.xml
 1 file added and 1 file fetched
 ```
+
+## Example: Transfer to cache
```

Comment:
Just clarifying since "to cache" can be a verb (but we mean the noun). Committing:
```suggestion
## Example: Transfer to the cache
```